packages/driver/test/cypress/integration/e2e/video_spec.js

```diff
@@ -0,0 +1,17 @@
+describe('video', () => {
+  it('play does not throw error on autoplay', () => {
+    cy.visit('/fixtures/video.html')
+    cy.get('video').then(($video) => {
+      return $video[0].play()
```

Comment:
No `.then()` or `.catch()` handler is required here because rejected promises will automatically cause the `cy.then` to fail.

Your notes are good but they can just go above the `.play()`